frontend/packages/kubevirt-plugin/integration-tests/tests/models/wizard.ts

```diff
@@ -97,13 +114,28 @@ export class Wizard {
   }
 
   async selectWorkloadProfile(workloadProfile: string) {
-    await selectOptionByText(view.workloadProfileSelect, workloadProfile);
+    await selectItemFromDropdown(view.workloadProfileSelect, view.dropDownItem(workloadProfile));
   }
 
-  async selectProvisionSource(provisionOptions) {
-    await selectOptionByText(view.provisionSourceSelect, provisionOptions.method);
-    if (Object.prototype.hasOwnProperty.call(provisionOptions, 'source')) {
-      await fillInput(view.provisionSourceInputs[provisionOptions.method], provisionOptions.source);
+  async disableGoldenImageCloneCheckbox() {
+    if (
+      (await view.goldenImageCloneCheckbox.isPresent()) &&
+      (await view.goldenImageCloneCheckbox.isSelected())
+    ) {
+      await click(view.goldenImageCloneCheckbox);
```

Comment:
I noticed on my env that some clicks (like next) dont always go through. Can you please add this to the click implementation?

`await browser.executeScript('arguments[0].scrollIntoView();', elem);`